shellfoundry_traffic/script_utils.py

```diff
@@ -19,8 +19,11 @@
 class ScriptCommandExecutor:
 
     def __init__(self, script_definition_yaml: str) -> None:
-        shell_definition_yaml = Path(os.getcwd()).joinpath(f'{script_definition_yaml}.yaml')
-        with open(shell_definition_yaml, 'r') as file:
+        if '.yaml' in script_definition_yaml:
+            script_definition_yaml_full_path = Path(os.getcwd()).joinpath(f'{script_definition_yaml}')
+        else:
+            script_definition_yaml_full_path = Path(os.getcwd()).joinpath(f'{script_definition_yaml}.yaml')
+        with open(script_definition_yaml_full_path, 'r') as file:
```

Comment:
Take a look at endswith - https://docs.python.org/3/library/stdtypes.html
Better than testing if str is in str.
Also, you can do it with fewer lines. I would do a one lines like:
script_definition_yaml = script_definition_yaml if CONDITION else f'{script_definition_yaml}.yaml'
And then you can build the Path with the modified script_definition_yaml.
Hope I'm clear...